src/onDatatransform/removeUnscheduledVisits.js

```diff
@@ -0,0 +1,6 @@
+export default function removeUnscheduledVists() {
```

Comment:
Seems like this should be in #76 - Is `config.unscheduled_visits` defined yet? No biggie though. I assume this will be used in the later PRs.

Reply:
Rightio.  Just copying stuff from the safety-results-over-time updates.